crates/bevy_app/src/plugin.rs

```diff
@@ -12,4 +12,4 @@ pub trait Plugin: Any + Send + Sync {
     }
 }
 
-pub type CreatePlugin = unsafe fn() -> *mut dyn Plugin;
+pub type CreatePlugin = fn() -> *mut dyn Plugin;
```

Comment:
This is unsafe. There is no guarantee that the plugin dylib was built against the same version of bevy, nor that it is written in rust at all. It may just as well be implemented as `void *_bevy_create_plugin() { return 0; }` in C.

Reply:
Well bloody my PR is going to change nothing. I'll leave this review open till I find something else to change